test/client-request.js

```diff
@@ -1,3 +1,5 @@
+/* globals AbortController */
```

Comment:
Does this not somehow inject `AbortController` for tests on older versions, i see this pattern being used in tests for eg: `test/fetch/client-fetch.js`

Reply:
no, it's just telling prettier that AbortController is a global

Reply:
Any ideas then @KhafraDev what could we do about this test?